src/utilcode/util.cpp

```diff
@@ -1306,10 +1303,8 @@ int GetCurrentProcessCpuCount()
 
 #else // !FEATURE_PAL
 
-    SYSTEM_INFO sysInfo;
-    ::GetSystemInfo(&sysInfo);
-    cCPUs = sysInfo.dwNumberOfProcessors;
-    return sysInfo.dwNumberOfProcessors;
+    cCPUs = PAL_GetCurrentProcessCpuCount();
```

Comment:
Could you please remove the ifdef instead and use the same code for Windows / Unix? And remove the PALGetCurrentProcessCpuCount from pal.h